front/app/containers/HomePage/SignedOutHeader/FixedRatioLayout.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import {
+  HeaderImage,
+  HeaderImageBackground,
+  HeaderImageOverlay,
+} from 'components/LandingPages/citizen/FullWidthBannerLayout';
+import {
+  Container,
+  Header,
+} from 'components/LandingPages/citizen/FixedRatioBannerLayout';
+import HeaderContent from './HeaderContent';
+
+import useHomepageSettings from 'hooks/useHomepageSettings';
+import { isNilOrError } from 'utils/helperUtils';
+
+const FixedRatioLayout = () => {
+  const homepageSettings = useHomepageSettings();
+
+  if (!isNilOrError(homepageSettings)) {
+    const headerImage = homepageSettings.attributes.header_bg?.large;
```

Comment:
Does this work as expected - displaying the component, the image and background as it should based on the BE data?